akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -247,3 +250,55 @@ private[akka] final class AcknowledgeSink[In](bufferSize: Int, val attributes: A
     new AcknowledgeSink[In](bufferSize, attr, amendShape(attr), timeout)
   override def toString: String = "AcknowledgeSink"
 }
+
```

Comment:
I am not sure the 3 classes below are needed, there is a one-liner that does the same using existing API as far as I understand.